addon/background-script.js

```diff
@@ -0,0 +1,12 @@
+/*console.log(document.querySelectorAll('iframe'))*/
+/* setTimeout(() => {
+  document.querySelector('iframe').src = "https://icatcare.org/app/uploads/2018/06/Layer-1704-1920x840.jpg"
+}, 1000) */
+let ad = document.querySelectorAll(iframe)
+for (let i = 0; i < ad.length; i++ ){
+  let father = ad[i].parentElement
+father.removeChild(ad[i])
+const img =
+father.appendChild('img.src = https://icatcare.org/app/uploads/2018/06/Layer-1704-1920x840.jpg')
```

Comment:
appendChild expects an actual element object as it's argument, not a string like this. so first u need to create an img element
```js
const img = document.createElement('img')
```
then u can change that img element's src:
```js
img.src = 'https://icatcare.org/app/uploads/2018/06/Layer-1704-1920x840.jpg'
```
&& only then can u append it to the parent element
```js
father.appendChild(img)
```